src/app/utils/deriveDatesAndCost.ts

```diff
@@ -50,8 +57,49 @@ export const deriveDatesAndCost = (trip: Trip) => {
 
 export const deriveDuration = (startDate: Date, endDate: Date) => {
   return (
-    Math.floor(
+    Math.ceil(
       Math.abs(endDate.getTime() - startDate.getTime()) / (1000 * 60 * 60 * 24)
     ) + 1
   );
 };
+
+export const exchange = (
+  costCurrency: string,
+  cost: number,
+  exchangeRates: ExchangeRatesDTO | null
+) => {
+  switch (costCurrency) {
+    case 'ZAR':
```

Comment:
Perhaps set the cost outside of the switch to the value of `cost` so you don't have to create a var per case.
i.e.
```ts
let outputValue = cost;
switch(costCurrency) {
  case 'ZAR':
    if (exchangeRates?.data?.['ZAR'].value) {
      outputValue = cost / exchangeRates.data['ZAR'].value;
    }
    return outputValue;
...
}
```